app/controllers/items.py

```diff
@@ -1,21 +1,136 @@
 from app import app, db
 from flask import Flask, request, jsonify
 from flask_sqlalchemy import SQLAlchemy
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
 from app.models.item import Item
 
-class ItemRoutes:
-    @app.route("/items")
-    def get_items():
-        try:
-            items=Item.query.all()
-            return jsonify([e.serialize() for e in items])
-        except Exception as e:
-            return(str(e))
-
-    @app.route("/items/<id_>")
-    def get_item(id_):
-        try:
-            item=Item.query.filter_by(id=id_).first()
-            return jsonify(item.serialize())
-        except Exception as e:
-            return(str(e))
+item_fields = {
+    'id': fields.Integer,
+    'title': fields.String,
+    'description': fields.String,
+    'price': fields.Float,
+    'donor': fields.String,
+    'status': fields.String,
+    'category': fields.String,
+    'charity': fields.String,
+    'image': fields.String
+}
+
+item_list_fields = {
+    'count': fields.Integer,
+    'items': fields.List(fields.Nested(item_fields))
+}
+
+item_post_parser = reqparse.RequestParser()
```

Comment:
Note: in the add argument statements below we've set each of these fields as required. I don't believe there are any situations where we would be OK with any of them being blank, but may want to review that with FE team.